crates/sargon/fixtures/transaction/create_access_controller_for_account.rtm

```diff
@@ -0,0 +1,98 @@
+CALL_METHOD
+    Address("account_rdx128dtethfy8ujrsfdztemyjk0kvhnah6dafr57frz85dcw2c8z0td87")
+    "securify"
+;
+TAKE_FROM_WORKTOP
+    Address("resource_rdx1nfxxxxxxxxxxaccwnrxxxxxxxxx006664022062xxxxxxxxxaccwnr")
+    Decimal("1")
+    Bucket("bucket1")
+;
+CREATE_ACCESS_CONTROLLER
+    Bucket("bucket1")
+    Tuple(
```

Comment:
@0xOmarA the `RuleSet` uses **position** (index) to determine which `AccessRule` is `primary`, `recovery` and `confirmation`? Because The values of the `Enum` seems to be the same?

```
        Enum<2u8>(
            Enum<1u8>(
                Array<Enum>(
```
for all three?

Reply:
It does indeed use the position to determine that since these are just arguments to the function.